### DISCUSSION TITLE
/help is not helpful [QOL]

### DISCUSSION THREAD

**Discussion by WildPenquin** [PaperMC/Paper#10595] [Ideas]
Hi,

When typing  `/help [any command]`, I do not get any kind of helpful usage summary. It seems like all usage information is missing or the help message is truncated. An example:
```
[12:10:28 INFO]: --------- Help: /whitelist ----------------------
[12:10:28 INFO]: Description: A Mojang provided command.
[12:10:28 INFO]: Usage: whitelist
```
In the official server one could get something useful out, though it is not prettily formatted:
```
/help whitelist
[12:23:29] [Server thread/INFO]: /whitelist on
[12:23:29] [Server thread/INFO]: /whitelist off
[12:23:29] [Server thread/INFO]: /whitelist list
[12:23:29] [Server thread/INFO]: /whitelist add <targets>
[12:23:29] [Server thread/INFO]: /whitelist remove <targets>
[12:23:29] [Server thread/INFO]: /whitelist reload
```

### Describe the solution you'd like.

Print out a helpful usage summary for all commands. Now one  needs to open a wiki page or something for all commands one does not remember how to use =).

### Describe alternatives you've considered.

Open a browser and use the wiki.

**Comment by electronicboy** [comment]
This is basically intended behavior in a sense, craftbukkit replaces the entire help system built into vanilla, which made a lot more sense when they also modified the command itself as they could also add the missing syntax information in a more traditional manner. outside of stripping that out, this is probably a "won't fix/WAI"

**Comment by codeHusky** [comment]
spigot could probably make some changes to the help system to actually respect the new help info and provide it somehow, but IMO probably better implemented via plugin developers if anything. would be helpful to have an API to expose the vanilla command's help info though
